docs/extend/modular-resources/configure.md

```diff
@@ -251,9 +295,9 @@ Add these properties to your module's configuration:
 {{% /tab %}}
 {{% /tabs %}}
 
-### Configure your modular resource
+### Configure a modular resource
 
-Once you have configured a module as part of your robot configuration, you can add any number of the resources that the module makes available to your robot by adding new components or services configured with your modular resources' [model](/extend/modular-resources/key-concepts/#models).
+Once you have added a local module to your robot, you can add any number of the {{< glossary_tooltip term_id="resource" text="resources" >}} provided by that module to your robot by adding new components or services that make use of your modular resources' [model](/extend/modular-resources/key-concepts/#models).
```

Comment:
```suggestion
Once you have added a local module to your robot, you can add any number of the {{< glossary_tooltip term_id="resource" text="resources" >}} provided by that module to your robot by adding new components or services that use your modular resources' [model](/extend/modular-resources/key-concepts/#models).
```
(opt) Think has to be super obvious that they use the model directly/that is what they are